glossary/contractor-definition.md

```diff
@@ -0,0 +1,8 @@
+---
+type: Resource
+---
+
+# Contractor
+
+Some roles in our organization are fulfilled by contractors.
+For transparency, these are mentioned with the legal entity they bill through.
```

Comment:
This is not useful in a definition, this is relevant context on the staff page only.

Reply:
@demkodo is fixing this in #774.